src/scripts/SpellHandlers/PaladinSpells.cpp

```diff
@@ -366,6 +366,17 @@ bool JudgementOfTheWise(uint32 i, Spell* s)
     return true;
 }
 
+bool GuardedByTheLight(uint32 i, Spell* s)
+{
+	if (!s->p_caster)
+		return false;
+
+	if (Aura* aura = s->p_caster->FindAura(54428))
```

Comment:
if (Aura* aura = s-p_caster->GetAuraWithId(54428))
should do the trick and resolve compiler errors